jsonrpc/eth_endpoint_test.go

```diff
@@ -105,6 +106,77 @@ func (m *mockBlockStore2) add(blocks ...*types.Block) {
 }
 
 func (m *mockBlockStore2) GetReceiptsByHash(hash types.Hash) ([]*types.Receipt, error) {
+	switch hash {
```

Comment:
Is it possible to set mock value when you use mockBlockStore2 in test?

Reply:
@Kourin1996 not exactly sure what do you mean by this?

Reply:
@lazartravica This function returns constant values actually. We need to write test case which matches these return value. So mock should be able to set value the mock returns in test code like as other mock if possible.